packages/ui/src/components/internal/Root.svelte

```diff
@@ -141,6 +141,13 @@
     $deviceInfo.isMobile &&
     (($deviceInfo.isPortrait && $deviceInfo.docWidth <= 480) ||
       (!$deviceInfo.isPortrait && $deviceInfo.docHeight <= 480))
+
+  $deviceInfo.mondayStart = localStorage.getItem('mondayStart') === 'true' ?? true
```

Comment:
Maybe we should save `mondayStart` in personal `Preferences` instead of  `localStorage`?

Reply:
Do we need this setting at all? Is that possible to figure out by locale?

Reply:
It seems that the best option is to get default by locale + setting if someone's preferences are different